### ISSUE TITLE
Suggestion: Upload consciousness, go from Android to AI

### ISSUE THREAD

**Issue by mm04926412** [simeonradivoev/MatterOverdrive#579]
tl:dr you can upgrade from one android to a hal 9000 style computers controlling many androids from the comfort of your fortified bunker and have a large amount of remote access capabilities to machines and inventories as long as there is a connection to your mainframe

LONG VERSION

In a lot of sci-fi settings there is a central ship AI that can control the ship remotely, I suggest adding this to matter overdrive as the next stage of evolution after becoming an android.

Becoming an AI would consist of creating an AI mainframe, which a maxed out ability android player could upload themselves to, where instead of the player being an android, the players actual body so to speak would now be inside the mainframe and require a lot more rf than a simple android would, the player would then remotely control different androids which could be built for specialised tasks and have the player be able to jump between the androids at will. For example a mining body would be weaker and a bit slower but could break blocks far faster you could throw that android down your mine and just leave it there with a charging station and a chest pumping it's resources back to the base, it would just stay in the mine dormant until you decide you need resources and switch to it, you could even have a combat android, technician android... maybe even a "human slave" to go back to normal for a bit, each option would cost a lot to produce and would have a specific upgrade path depending on it's specification. Other non playable machines such as spider drones (Read Thaumcraft Golems) could be given commands remotely such as deliver item x from mainframe connected inventory y to android z,

In this way an end game player would not simply be living in their base, but become their base and having an army of controllable bodies and non playable utility workers would add an interesting new almost RTS layer to minecraft that no mod has ever done before.

At the most extreme end of this idea the AI could even have access to a top down map of the local area kind of like dwarf fortress and build factories by commanding worker drones to place block x in location y and have production quotas etc.

**Comment by shadowfacts** [comment]
1. Holy wall of text, batman.
2. Holy out of scope idea, batman.
3. 

> kind of like dwarf fortress

So you want an entire massive game inside of a Minecraft mod?

**Comment by mm04926412** [comment]
well the only real new mechanics were

You have several specialised android bodies you can switch between with a menu with a differing upgrade path depe

you have a journeymap looking representation of your base, you can tell thaumcraft golem like things to do one off manual tasks with it.

I appreciate the wall of text made it look like a massive endeavor but dwarf fortress was brought up for camera perspective reasons only.

**Comment by shadowfacts** [comment]
From a technical standpoint, it's still massively complicated.
